### PR TITLE
Group imports with rustfmt

### PR DESCRIPTION
**Pull request by Nutomic**
There was some inconsistency in imports because sometimes an empty line sneaked between import lines, and cargo fmt didnt remove it. This setting guarantees that all imports are merged into a single block. Value StdExternalCrate could also work but doesnt make much sense to me.

https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=import#group_imports

### CODE REVIEW

File: .rustfmt.toml [hunk 1]
```diff
@@ -2,4 +2,4 @@ tab_spaces = 2
 edition="2021"
 imports_layout="HorizontalVertical"
 imports_granularity="Crate"
-reorder_imports=true
```
Comment (Nutomic) [inline .rustfmt.toml:5]: We can just rely on the default value for this:

https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=import#reorder_imports

File: crates/db_schema/src/source/comment.rs [hunk 2]
```diff
@@ -1,11 +1,10 @@
 use crate::newtypes::{CommentId, DbUrl, LanguageId, LtreeDef, PersonId, PostId};
+#[cfg(feature = "full")]
+use crate::schema::{comment, comment_like, comment_saved};
```
Comment (Nutomic) [inline crates/db_schema/src/source/comment.rs:3]: This isnt so nice, it doesnt keep imports with feature declaration in a separate block.

Comment (dessalines) [reply crates/db_schema/src/source/comment.rs:3]: Not much we can do about that, oh well.

**Comment by dessalines** [comment]
Another one that's important, but that rustfmt can't do, is removing wildcard imports: https://github.com/rust-lang/rustfmt/issues/5569

I removed a ton of those when I did diesel_async, but I'm sure there are some that I missed.

**Review by dessalines — APPROVED** [review]

**Comment by dessalines** [comment]
Hrm cargo check failed.

**Comment by dessalines** [comment]
I'm investigating why this one's failing.

**Comment by dessalines** [comment]
Closing in favor of https://github.com/LemmyNet/lemmy/pull/2526

**Pull request closed without merging**